### PR TITLE
[AAA] Change mint stat calculations for payment type

### PR DESCRIPTION
**Pull request by DidacSF**
## Description

Changes stat calculations for minting so that free mints and normal mints got to different sections, instead of free mints also counting as normal mints.

## Type of changes

- [ ] `build`: Changes that affect the build system or external dependencies (eg, Cargo, Docker)
- [ ] `ci`: Changes to CI configuration
- [ ] `docs`: Changes to documentation only
- [ ] `feat`: Changes to add a new feature
- [x] `fix`: Changes to fix a bug
- [ ] `refactor`: Changes that do not alter functionality
- [ ] `style`: Changes to format the code
- [ ] `test`: Changes to add missing tests or correct existing tests

## Checklist

- [ ] Tests for the changes have been added
- [ ] Necessary documentation is added (if appropriate)
- [ ] Formatted with `cargo fmt --all`
- [ ] Linted with `cargo clippy --all-features --all-targets`
- [ ] Tested with `cargo test --workspace --all-features --all-targets`

### CODE REVIEW

File: pallets/ajuna-awesome-avatars/src/tests.rs
Comment (clangenb) [inline pallets/ajuna-awesome-avatars/src/tests.rs:1]: Looks good to me, but I just think that this test is very long and hard to follow. So I have one question and one comment.

1. Do we expect different behavior here for different mint batch sizes?
2. I would recommend rewriting this test with [rstest](https://crates.io/crates/rstest) and parametrize this test to make it more readable and reduce the amount of written code.

**Review by clangenb — APPROVED** [review]
Looks good to me! I just have a comment and a question which is not directly related to the changes in this PR!

File: pallets/ajuna-awesome-avatars/src/tests.rs
Comment (DidacSF) [reply pallets/ajuna-awesome-avatars/src/tests.rs:1]: 1. Not really
2. I would prefer not to introduce dependencies for testing

**Review by darkfriend77 — APPROVED** [review]
okay
